### DISCUSSION TITLE
OpenPose + Canny to create image

### DISCUSSION THREAD

**Discussion by dayunbao** [Mikubill/sd-webui-controlnet#1853] [Q&A] [answered]
I'm trying to use a multi-ControlNet with OpenPose and Canny Edges.  I fed an image of an apartment to the Canny Edge preprocessor, and was hoping I could "layer" the OpenPose skeletons on top of it to create figures based on the OpenPose skeletons in the apartment.  However, it hasn't worked, so far.  Is there something I'm doing wrong?  I don't know if it matters, but ControlNet 0 used the OpenPose preprocessor, while ControlNet 1 used Canny Edges.  The prompt I used was "Zombie attacking a woman in an apartment at night."

If you look very carefully, you can kind of see parts of the "zombie" mixed into the image of the apartment. 

See screenshots from  the Automatic1111 web UI below.  I have gotten similarly weird results from my own app, as well.

![openpose](https://github.com/Mikubill/sd-webui-controlnet/assets/11196050/5cfdb408-2dab-4410-bd48-1afc90150063)
![canny](https://github.com/Mikubill/sd-webui-controlnet/assets/11196050/cb35cae5-c62c-4129-bf11-f8d9671d2951)
![grid](https://github.com/Mikubill/sd-webui-controlnet/assets/11196050/713cc4fd-b04a-4fa8-be48-4034c8c0b289)

**Comment by huchenlei** [comment]
See https://github.com/Mikubill/sd-webui-controlnet/issues/1736. 

Is that what you want? https://github.com/huchenlei/stable-diffusion-ps-pea. You can join the [discord server](https://discord.gg/GkaWcUat7R) for further assistance.
![img](https://user-images.githubusercontent.com/20929282/256951742-5dcb6d6f-5c3e-4cf8-abf6-c5223059a8af.png)
![img](https://user-images.githubusercontent.com/20929282/256952066-1938eac3-4a11-4d6a-95c1-eda23be453ea.png)

**Comment by huchenlei** [comment, marked as answer]
Offline discussion:
![Screen Capture 015 - Photopea - Online Photo Editor - www photopea com](https://github.com/Mikubill/sd-webui-controlnet/assets/20929282/f51458ab-b342-4c6d-a102-b0c280cb893a)

![New Project](https://github.com/Mikubill/sd-webui-controlnet/assets/20929282/0139fac9-8bc8-41b8-ad42-8a14ca6aef19)

**Reply by dayunbao** [reply]
This gives me an idea of what to do.  Thanks, and sorry for accepting this as the answer so late!
